backend/fms_core/services/process_measurement.py

```diff
@@ -15,7 +15,7 @@ def create_process_measurement(process, source_sample, execution_date, volume_us
     if not source_sample:
         errors.append(f"Source sample is required for process measurement creation.")
     if not execution_date:
-        errors.append(f"Execution date is required for process measurement creation.")
+        errors.append(f"Execution date is required for process measurement creation. Format expected: YYYY-MM-DD")
```

Comment:
I would wrap this together in a single sentence, "An execution date with the format YYYY-MM-DD must be provided for process measurement creation."

Reply:
Good call